task/hadolint/0.1/README.md

```diff
@@ -0,0 +1,212 @@
+# Hadolint
+
+Tekton Task for hadolint https://github.com/hadolint/hadolint
+
+<img align="right" alt="pipecat" width="150"
+src="https://hadolint.github.io/hadolint/img/cat_container.png" />
+
+A smarter Dockerfile linter that helps you build [best practice][] Docker
+images. The linter parses the Dockerfile into an AST and performs rules on
+top of the AST. It stands on the shoulders of [ShellCheck][] to lint
+the Bash code inside `RUN` instructions.
+
+## Install the Task
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/hadolint/0.1/hadolint.yaml
+```
+
+## Pre-requisite
+Install git-clone task from catalog
+```
+https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.3/git-clone.yaml
+```
+
+## Workspaces
+* **source** : A Workspace containing your source directory.
+
+## Parameters
+* **ignore-rules** : A rule to ignore. Comma separated list of rule codes
+* **dockerfile-path** : path to Dockerfile.
+* **output-format** : The output format for the results [tty | json | checkstyle | codeclimate
+        | gitlab_codeclimate | codacy] (default tty).
+
+## Platforms
+
+The Task can be run on `linux/amd64` platform.
+
+## Usage
+
+```yaml
+---
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: hadolint-pipeline
+spec:
+  params:
+    - name: repository
+      type: string
+    - name: ignore-rules
+      type: string
+    - name: dockerfile-path
```

Comment:
similarly here. Probably you can remove this YAML from here and just add a reference to `run.yaml` from tests directory. Maybe you can take some reference from [`aws-cli`](https://github.com/tektoncd/catalog/tree/main/task/aws-cli/0.2#usage) task.

Reply:
Fixed.